### PR TITLE
Fix 33714 34095

### PR DESCRIPTION
**Pull request by miyadav**
Closing previous PR - https://github.com/openshift/verification-tests/pull/1763

Validations for both passed 

@jhou1  @sunzhaohua2  , have a look when time permits

### CODE REVIEW

File: features/machine/machine_healthcheck.feature [hunk 1]
```diff
@@ -271,14 +271,14 @@ Feature: MachineHealthCheck Test Scenarios
...
+      | maxUnhealthy: Invalid value: ".*": spec.maxUnhealthy |
 
    Examples:
       | maxUnhealthy |
       |  -2a         |
-      | "10t%"       |
-      | "-2%"        |
+      |  10t%        |
+      |  -2%         |
       |  -2%         |
```
Comment (liangxia) [inline features/machine/machine_healthcheck.feature:282]: The above two lines is the same ?

Comment (miyadav) [reply features/machine/machine_healthcheck.feature]: Thanks @liangxia  , removed it , squashed commits

**Pull request merged**